brave/brave-core
# Feature/playlist dangling 1.68.x

- Cleanup Playlist only every 7 days.
- Add Debug View for Playlist.


Uplift of: https://github.com/brave/brave-core/pull/24260
Resolves: https://github.com/brave/brave-browser/issues/39131



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Create 5,000 Playlist Items with 4990 dangling
- Check app size/space usage is huge
- Reset the 7 days cleanup
- Restart the app
- Check app size/space usage went down
